component-set/components/scripts/mathml.js

```diff
@@ -0,0 +1,34 @@
+(function(){
+    // Load the MathJax library
+    const script = document.createElement('script');
+    script.src = 'https://cdn.jsdelivr.net/npm/mathjax@3/es5/mml-chtml.js';
+    document.head.append(script)
+
+    
+    document.readyState === 'complete' ? onLoad() : window.addEventListener('load', onLoad);
+
+    function onLoad() {
+        // Perform MathJax typesetting when a MATH element is added
+        // Note that loading the library performs a one time parse of all elements.
+        // So for the editor we need to manually call MathJax.typeset() to parse the elements.
+        const observer = new MutationObserver(function(mutationsList) {
+            for (const mutation of mutationsList) {
+                for (const addedNode of Array.from(mutation.addedNodes)) {
+                    if (addedNode.nodeType !== Node.ELEMENT_NODE) {
+                        continue;
+                    }
+                    // Already processed by mathjax
+                    if (['MJX-CONTAINER'].includes(addedNode.tagName.toUpperCase())) {
```

Comment:
Could just be `addedNode.tagName.toUpperCase() === 'MJX-CONTAINER'`

Reply:
I thought I would make it future proof by allowing an array of nodes for which it should skip the check :)

Reply:
Fair enough, makes sense if the library is experimental 👍

Reply:
I mean, it might be nicer if we provided an api from the digital editor to re-trigger custom processing logic.